frontend/src/App.js

```diff
@@ -55,10 +64,26 @@ function App() {
           <p>Left Foot: {stepTime.targetZones.left.min} - {stepTime.targetZones.left.max}</p>
           <p>Right Foot: {stepTime.targetZones.right.min} - {stepTime.targetZones.right.max}</p>
           <p>Average Target Zone: {stepTime.targetZones.average}</p>
+        </div>
```

Comment:
We may want to have this just be as a console message or alert rather than a visualization on the page because it could affect other views. Additionally, it's at the bottom of the page so it might not been seen by the client anyway based on the view. We could leave it as is, just something to consider.